Projects/UOContent/Misc/Guild.cs

```diff
@@ -1562,7 +1554,33 @@ public void CalculateGuildmaster()
                 winner = m_Leader;
             }
 
-            if (m_Leader != winner && winner != null)
+            if (m_Leader == null && winner == null)
+            {
+                if (votes.Count > 0)
+                {
+                    var randomNumber = Utility.Random(votes.Count);
+                    var index = 0;
+                    foreach (var m in votes.Keys)
+                    {
+                        if (index++ == randomNumber)
+                        {
+                            winner = m;
+                            break;
+                        }
+                    }
+                }
+
+                if (winner == null && Members.Count > 0)
+                {
+                    winner = Members.RandomElement();
+                }
+            }
+
+            if (winner == null)
+            {
+                m_Leader = null;
+            }
+            else if (m_Leader != winner)
```

Comment:
If no leader was chosen, pick one at random from anyone that got votes, and if nobody got votes, choose one at random from all members, otherwise the guild leader is null, resulting in disbanding.